pkg/migration/controllers/clusterpair.go

```diff
@@ -8,11 +8,11 @@ import (
 
 	"github.com/libopenstorage/stork/drivers/volume"
 	stork "github.com/libopenstorage/stork/pkg/apis/stork"
-	storkv1 "github.com/libopenstorage/stork/pkg/apis/stork/v1alpha1"
+	storkv1alpha1 "github.com/libopenstorage/stork/pkg/apis/stork/v1alpha1"
```

Comment:
Let's not use alpha1 in the import name. If we update the version we'll need to update the references everywhere.

Reply:
done. Renamed the import to stork_api